samples/flexibleEngine-K8S.json

```diff
@@ -7,5 +7,6 @@
   "region": "eu-west-0",
   "customerServiceName": "default-k8s",
   "serviceHostingType": "self",
-  "serviceRequestProperties": {}
+  "serviceRequestProperties": {},
+  "serviceProviderContactDetails": {}
```

Comment:
Please remove this from all the JSON files. It must be added only to the service template yaml files.